src/gui/categoryfilterproxymodel.h

```diff
@@ -47,4 +49,6 @@ class CategoryFilterProxyModel final : public QSortFilterProxyModel
 private:
     // we added another overload of index(), hence this using directive:
     using QSortFilterProxyModel::index;
+
+    Utils::Compare::NaturalLessThan<Qt::CaseInsensitive> m_naturalLessThan;
```

Comment:
I wonder if it's really so hard to create that it makes sense to cache it like this?

Reply:
even if the creation is cheap, here the comparison could be invoked many (thousands or more) times and the creation overhead could still build up within a short time interval.